src/types/character/stat.ts

```diff
@@ -0,0 +1,5 @@
+export type Stat = {
+  name: 'Health Points' | 'Ætheris Points' | 'Strength' | 'Dexterity' | 'Defense' | 'Reflexes' | 'Senses' | 'Charisma' | 'Mind' | 'Spirit' | 'Willpower' | 'Luck' ,
```

Comment:
Better to create an enum like StatNamesEnum or StatNameEnum

Reply:
Will do!